### PR TITLE
2Dproperties

### PR DESCRIPTION
**Pull request by planetmariana**
This PR includes the implementation of the 2D lookup tables in a regular grid for different physical properties up to 135GPa. The main changes are how the function LookupProperty2D is being read from the tables following P,T, in this case the melting curves. The profiles for the three different phase cases looks like this:

**Solid phase**
solid
**Liquid phase**
liquid
**Mixed phase**
mixed_phase

### CODE REVIEW

**Comment by djbower** [comment]
This looks like it breaks a few tests and also introduces custom path names based on a local setup.  @planetmariana you probably need to tidy up this branch locally first, ensuring that existing tests don't break and that custom path setups are generalized, and then reissue the PR.

**Comment by timlichtenberg** [comment]
The tests seem to pass now, nice! There are still some changes to the EOS files; they can probably also be reverted?

File: data/1TPa-dK09-elec-free/MgSiO3_Wolf_Bower_2018/adiabat_temp_grad_melt.dat
Comment (timlichtenberg) [inline data/1TPa-dK09-elec-free/MgSiO3_Wolf_Bower_2018/adiabat_temp_grad_melt.dat:1]: This file here for example.

**Comment by planetmariana** [comment]
thanks @djbower , I did the changes you suggested. Could you please review it again ?

File: data/1TPa-dK09-elec-free/MgSiO3_Wolf_Bower_2018/adiabat_temp_grad_melt.dat
Comment (planetmariana) [reply data/1TPa-dK09-elec-free/MgSiO3_Wolf_Bower_2018/adiabat_temp_grad_melt.dat:1]: Thanks @timlichtenberg, already delete them.

**Review by djbower — APPROVED** [review]

**Pull request merged**